### PR TITLE
[CARBONDATA-798] Added support to rename bad records folder

### PR DESCRIPTION
**Pull request by kunal642**
Added support to rename Bad Records folder to new table name while renaming table.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1250/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1255/

File: integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/AlterTableCommands.scala [hunk 1]
```diff
@@ -198,6 +200,27 @@ private[sql] case class AlterTableRenameTable(alterTableRenameModel: AlterTableR
...
     Seq.empty
   }
+
+  private def renameBadRecords(oldTableName: String,
+      newTableName: String,
+      dataBaseName: String) = {
+    val badRecordsLocation = CarbonProperties.getInstance()
+      .getProperty(CarbonCommonConstants.CARBON_BADRECORDS_LOC,
+        CarbonCommonConstants.CARBON_BADRECORDS_LOC_DEFAULT_VAL)
+    val badRecordsDatabase = badRecordsLocation + CarbonCommonConstants.FILE_SEPARATOR +
```
Comment (gvramana) [inline integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/AlterTableCommands.scala]: Use a common function to get bad records location

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1272/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1274/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1275/

**Comment by kunal642** [comment]
@gvramana Changes have been done

**Comment by gvramana** [comment]
LGTM

**Pull request closed without merging**